### DISCUSSION TITLE
After accepting a package invite, take me to package page

### DISCUSSION THREAD

**Discussion by ljharb** [npm/feedback#902] [Website]
Currently, it dumps me on the homepage with a flash message - however, literally every single time, the place i next have to/want to go is the package page itself. Can you just redirect me there instead of the homepage?

**Comment by monishcm** [comment]
Thanks for the feedback, will put this in the backlog but it is going to take some time to prioritise this.

**Reply by ljharb** [reply]
Understood, altho it's probably a one-line change :-)

**Reply by monishcm** [reply]
Agree it should be a minor change. :)
